src/main/java/com/parsing/parsers/pdf/parsing/PageOCRPreparator.java

```diff
@@ -38,26 +41,40 @@ public class PageOCRPreparator {
     private static final String DIR_TO_READ_TESSDATA = "/tessdata/";
     private static final int MINIMAL_WIDTH_WORD_FOR_OCR = 3;
     private static final int THICKNESS_LINE = 5;
-    private static final double OFFSET = 5;
     private static final String REGEX = ".*[А-ЩЬЮЯЄЇІа-щьюяєїі].*";
     private static final double[] RGB_WHITE_COLOUR = {255, 255, 255};
 
-    public String preparePage(PDDocument document, int page) throws IOException {
-        log.debug("Class PageOCRPreparator started.");
-        String fileResult = "preparePage" + page + ".png";
-        String pagePDF = getPagePDF(document, page);
-        String rotatedPage = rotationImage.rotate(pagePDF);
-        Rectangle pageRectangle = getPageMainRectangle(document, page);
+    static final double HORIZONTAL_LINE_LENGTH = 700;
+    static final int HORIZONTAL_THRESHOLD = 5;
+    static final double ALL_LINES_THRESHOLD1 = 50;
+    static final double ALL_LINES_THRESHOLD2 = 200;
```

Comment:
what difference between 1 and 2?

Reply:
It's from opencv library, Canny's recommendation.
Params:
 threshold1 – first threshold for the hysteresis procedure.
threshold2 – second threshold for the hysteresis procedure.